docs/changelog/0.8.1.rst

```diff
@@ -370,9 +370,7 @@ Improved Documentation
 - Updated deprecated meeting and calendar links in
   :file:`README.md`. (`#1327
   <https://github.com/plasmapy/plasmapy/pull/1327>`__)
-- Enabled the `sphinx-hoverxref
-  <https://sphinx-hoverxref.readthedocs.io>`_ extension to
-  |Sphinx|. (`#1353
+- Enabled the ``sphinx-hoverxref`` extension to |Sphinx|. (`#1353
```

Comment:
```suggestion
- Enabled the `sphinx-hoverxref 
<https://github.com/readthedocs/sphinx-hoverxref>`_ extension to |Sphinx|. (`#1353
```

I think leaving a link to the archived repo would be good just to preserve link back.